notebooks/templates/Inspection_jobs_analysis_TEMPLATE.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<p><code>logger = logging.getLogger()</code> &lt;- duplicate line</p><p> </p><p>I would suggest using a logger name, in Jupyter, this might colide with default Jupyter logging.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>